src/Components/Artist/MarketInsights.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+
+export interface MarketInsightsProps extends RelayProps, React.HTMLProps<MarketInsights> {}
+
+const Categories = {
+  "blue-chip": "Blue Chip",
+  "top-established": "Top Established",
+  "top-emerging": "Top Emerging",
+}
+
+export class MarketInsights extends React.Component<MarketInsightsProps, null> {
+  renderGalleryRepresentation() {
+    const { highlights } = this.props.artist
+    const { partner_artists } = highlights
+    if (partner_artists && partner_artists.length > 0) {
+      const partnerArtist = partner_artists[0]
+      const { categories } = partnerArtist.partner
+      let category
+      Object.keys(Categories).forEach(key => {
+        categories.forEach(partnerCategory => {
+          if (partnerCategory.id === key) {
+            category = Categories[key]
+          }
+        })
+      })
+      return (
+        <div>
+          Represented by {this.props.artist.highlights.partner_artists[0].partner.name}
```

Comment:
I should make this more defensive, not rendering if there are no `highlights` or `partner_artists`.